standard.mk

```diff
@@ -0,0 +1,3 @@
+.PHONY: coverage
```

Comment:
We do not need a separate Makefile for a single target, instead let's add the `coverage` target to the existing Makefile in the project root.

Reply:
Token has not been generated at yes as I wasn't sure if this PR needed merged first.  I can do that part.

The .gitignore file was already ignoring *.out files, so I did not change it.

and I have added the coverage target to the existing makefile.  i have tried to follow the existing style, please let me know if it isn't correct.